packages/core/admin/admin/src/core/utils/axiosInstance.js

```diff
@@ -8,11 +8,14 @@ const instance = axios.create({
 instance.interceptors.request.use(
   async config => {
     config.headers = {
-      Authorization: `Bearer ${auth.getToken()}`,
       Accept: 'application/json',
       'Content-Type': 'application/json',
     };
 
+    if (auth.getToken() !== null) {
+      config.headers.Authorization = `Bearer ${auth.getToken()}`;
+    }
+
     return config;
   },
```

Comment:
What about using spread operator to keep it simple?
```suggestion
      ...(auth.getToken() && { Authorization: `Bearer ${auth.getToken()}` }),
      Accept: 'application/json',
      'Content-Type': 'application/json',
    };

    return config;
  },
```